src/Components/Common/Avatar.tsx

```diff
@@ -34,6 +34,9 @@ const toColor = (name: string): [string, string] => {
 };
 
 const initials = (name: string): string => {
+  if (!name) {
+    return ""; // Return empty string or a default value if the name is undefined or null
+  }
```

Comment:
When would this happen 🤔

Reply:
At a certain point, I encountered an issue, so i added it. But now its fine. I will remove it.